.flowconfig

```diff
@@ -31,6 +31,7 @@ module.name_mapper='^rewards\(.*\)$' -> '<PROJECT_ROOT>/ui/rewards\1'
 module.name_mapper='^i18n\(.*\)$' -> '<PROJECT_ROOT>/ui/i18n\1'
 module.name_mapper='^effects\(.*\)$' -> '<PROJECT_ROOT>/ui/effects\1'
```

Comment:
When I selected "Both", the sidebar does change, but the homepage section titles still remained in English until I manually refreshed

Reply:
I noticed it too, and if logged in the header "Your Wallet" also doesn't change, but when language selected from the settings page it does instantly, I wonder what's it doing differently? will investigate

Reply:
[#972 Render whole app on language change](https://github.com/OdyseeTeam/odysee-frontend/pull/972) might help to fix this.